config/custom-environment-variables.js

```diff
@@ -87,4 +87,9 @@ module.exports = {
   routesCacheTTL: {
     "/members": "ROUTESCACHETTL_MEMBERS",
   },
+
+  Event100ms: {
+    APP_ACCESS_KEY: "APP_ACCESS_KEY",
+    APP_SECRET: "APP_SECRET",
```

Comment:
`APP_ACCESS_KEY` and `APP_SECRET` is very general
can we change this to something like `EVENT_100MS_APP_ACCESS_KEY` and 
`EVENT_100MS_APP_SECREt`
or to something else